src/engine.ts

```diff
@@ -337,12 +337,9 @@ function parseIdentifier(p: AdhocParser): string {
   return match[0];
 }
 
-// This function does stuff like `min-width` => `MinWidth`
+// This function does stuff like `min-width` => `minWidth`
```

Comment:
I use this as a lookup for the enum, so that’s why I had the first letter capitalized :D